README.md

```diff
@@ -200,4 +201,14 @@ NULL ...
 ### Internals
 ![](./assets/internals.png)
 
+## Versioning
```

Comment:
This section seems better in CONTRIBUTING.md. The README is targeted at users who don't care about this. (It's already covered above in the statement "Use the version of django-spanner that corresponds to your version of Django.
For example, django-spanner 2.2.x works with Django 2.2.y. (This is the only supported version at this time.)")